App/proguard-rules.txt

```diff
@@ -34,9 +34,24 @@
 -dontusemixedcaseclassnames
 -dontskipnonpubliclibraryclasses
 -verbose
+-dontobfuscate
+-forceprocessing
+
+-keep class * extends android.app.Activity
+-assumenosideeffects class android.util.Log {
+    public static *** d(...);
+    public static *** v(...);
+    public static *** i(...);
+    public static *** w(...);
+    public static *** e(...);
+    public static *** wtf(...);
+}
+
+#Building with optimization enabled fails without this.
+-optimizations !code/allocation/variable
 
 # We restrict a few more optimizations for the maps library.
--optimizations !code/simplification/arithmetic,!field/*,!class/merging/*,!code/simplification/variable
+#-optimizations !code/simplification/arithmetic,!field/*,!class/merging/*,!code/simplification/variable
```

Comment:
If this needs to be removed, it's better to actually remove it (and the associated comment) from the file. The lines are always available in the version control history.